source/sensdet/PetSensorHit.h

```diff
@@ -15,96 +15,88 @@
 #include <G4ThreeVector.hh>
 
 
-namespace nexus {
```

Comment:
In this case `namespace nexus` is not required because  the class `PetSensorHit` is from `petalosim`, right?

Reply:
Yes, the namespace thing is just for convenience. There's no point in including petalosim classes in the `nexus` namespace.